content/chat/getting-started/javascript.textile

```diff
@@ -0,0 +1,195 @@
+---
+title: "Getting started: Chat and JavaScript"
+meta_description: "A getting started guide for Ably Chat that steps through some of the key features using JavaScript."
+meta_keywords: "Ably, realtime, quickstart, getting started, basics, chat"
+---
+
+This guide will get you started with Ably Chat.
+
+It will take you through the following steps:
+
+* Complete the prerequisites before you begin.
+* Create a client and establish a realtime connection to Ably.
+* Create a room and subscribe to its messages.
+* Send a message to the room, and then edit that message.
+* Set up a typing indicator to see which clients are typing.
+* Display the online status of clients to the room.
+* Subscribe and send reactions to the room.
+* Retrieve historical messages to provide context to new joiners.
+
+h2(#prerequisites). Prerequisites
+
+* Sign up for an Ably account.
+* Create a new app, and get your first API key.
+** Your API key will need the @publish@, @subscribe@, @presence@ and @history@ capabilities.
+* Install the Ably CLI: @npm install -g &#64;ably/cli@
+* Run @ably login@ in the terminal and follow the prompts to provide an access token to the Ably CLI.
+* Set the default app to use with @ably apps switch@ and select an API key to use for operations with @ably auth keys switch@.
+* Install the Ably Chat JavaScript SDK in a new project within your IDE. This will also install the Ably Pub/Sub SDK as it is a dependency:
```

Comment:
It won't install it automatically, `ably` is `peerDependency` meaning that the user must manually install it.

Reply:
Can we mention that `react` being required for now is a **temporary** thing that we intend to fix?

Reply:
I believe that peer dependencies are installed by default in newer versions of NPM now. 

Added a note about `react` - good point.

Reply:
Yep you're right re peer